addons/l10n_eu_service/wizard/wizard.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import fields, models, api
+from openerp.tools.translate import _
+
+class l10n_eu_service(models.TransientModel):
+
+    """Create fiscal positions for EU Service VAT"""
+
+    _name = "l10n_eu_service.wizard"
+    _description = __doc__
+
+    def compute_default_value_chart(self):
```

Comment:
Move these below fields, prefix them with `_` so they can't be called over RPC (once we eventually add a new-api-rpc)